ci-scripts/request-translation.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+cd ..
+
+export SLACK_CHANNEL='#doctools-publish'
+export targets=( "oce" "asa" "eu" "oie" "wf" "oag" )
```

Comment:
Do you see a drawback with using uppercase rather than lowercase, for prod names?
Rationale: I think it's a little easier to read, among a list of branches or projects.
(not a blocker by any means)

Reply:
Branch name has target name in upper case
<img width="750" alt="image" src="https://user-images.githubusercontent.com/4158731/202065436-cd4b60c5-5998-4a08-9f0e-a595d0bdc1df.png">

the trick is double `^` when expanding variable 
```
export TOPIC_BRANCH="docs_l10n_request_${TARGET^^}_$(date +"%s")"
```